include/index.h

```diff
@@ -104,6 +104,9 @@ template <typename T, typename TagT = uint32_t, typename LabelT = uint32_t> clas
     DISKANN_DLLEXPORT size_t get_num_points();
     DISKANN_DLLEXPORT size_t get_max_points();
 
+    DISKANN_DLLEXPORT const std::vector<LabelT> find_common_filters(uint32_t point_id, bool search_invocation,
```

Comment:
Might be better to return the common filters by reference and return the count as return type.

Reply:
Common filters vector is also not getting used further. so I have just initialized it inside the find_common_filters methos. Just returned the size of this vector